pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -526,7 +526,7 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		}
 	}
 
-	mountOptions = append(mountOptions, "dax")
+	mountOptions = append(mountOptions, "dax=always")
```

Comment:
Does that increase our minimum supported kernel version?

Reply:
That's a valid concern. If we do not want to change mount option, we have to loosen our options compare function to accept 'dax' matching to 'dax=always'. I was thinking about  that, too. It could be done, for example either writing special handling for dax=always, or trying to be more generic and strip always part with equal sign so that we treat option itself as significant part and value as stripped part